src/material-experimental/theming/_m3-density.scss

```diff
@@ -63,7 +63,13 @@ $_density-tokens: (
     header-expanded-state-height: (64px, 60px, 56px, 48px),
   ),
   (mat, fab): (),
-  (mat, form-fild): (),
+  (mat, form-field): (
+    container-height: (56px, 52px, 48px, 44px, 40px, 36px),
+    filled-label-display: (block, block, none, none, none, none),
+    container-vertical-padding: (16px, 14px, 12px, 10px, 8px, 6px),
+    filled-with-label-container-padding-top: (24px, 22px, 12px, 10px, 8px, 6px),
+    filled-with-label-container-padding-bottom: (8px, 6px, 12px, 10px, 8px, 6px),
```

Comment:
OOC: why are there 6 values, if there are only 0-4 levels of density?

Reply:
I think it’s also for minimum and maximum. I just copied over the values from the current density since it’s the same as the M2 form field.

Reply:
Ahh, would be good if the docs for this map would describe that better. I have no idea where minimum and maximum are stored.

Reply:
In M2 they were coming from some hardcoded map in MDC. In M3 we need to hardcode them because of custom padding approach.

Reply:
Sounds reasonable. I think the comment for this map should reflect this somehow, but unrelated to this PR